eng/helix/helix.proj

```diff
@@ -23,13 +23,30 @@
     <IsExternal>true</IsExternal>
     <MaxRetryCount Condition="'$(MaxRetryCount)' == ''">2</MaxRetryCount>
     <HelixAccessToken Condition="'$(_UseHelixOpenQueues)' != 'true'">$(HelixApiAccessToken)</HelixAccessToken>
+    <IncludeDotNetCli>true</IncludeDotNetCli>
+    <DotNetCliPackageType>sdk</DotNetCliPackageType>
+    <DotNetCliVersion>$(NETCoreSdkVersion)</DotNetCliVersion>    
+    <DotNetCliChannel>Current</DotNetCliChannel>
 
     <!-- Copied from Microsoft.NET.DefaultOutputPaths.targets in the .NET SDK. The Helix SDK contains nothing similar. -->
     <BaseOutputPath Condition="'$(BaseOutputPath)' == ''">bin\</BaseOutputPath>
     <OutputPath Condition="'$(OutputPath)' == '' and '$(PlatformName)' == 'AnyCPU'">$(BaseOutputPath)$(Configuration)\</OutputPath>
     <OutputPath Condition="'$(OutputPath)' == '' and '$(PlatformName)' != 'AnyCPU'">$(BaseOutputPath)$(PlatformName)\$(Configuration)\</OutputPath>
   </PropertyGroup>
 
+  <!-- Specify the runtime we need which will be included as a correlation payload -->
+  <ItemGroup>
+    <!--
+      When the targeting pack builds, it has exactly the same version as the shared framework. Passing
+      SharedFxVersion because that's needed even when the targeting pack isn't building.
+      Use the BrowserDebugHost transport package as a sentinel for the non-shipping version of the NETCoreApp shared framework.
+    <AdditionalDotNetPackage Include="$(MicrosoftNETCoreBrowserDebugHostTransportVersion)">
+    -->
+    <AdditionalDotNetPackage Include="6.0.0-preview.3.21167.1">
+      <PackageType>runtime</PackageType>
+    </AdditionalDotNetPackage>
+  </ItemGroup>
```

Comment:
Remove this and uncomment the above.

Reply:
I'm just going to get the helix side of things working first, I'll worry about our msbuild fun after all the tests are passing with a hardcoded version that I know works.